short/index.html

```diff
@@ -248,6 +236,18 @@
 </section>
 
 <section data-markdown>
+**Code Review**
+
+*   Academics review each other's papers; what about code?
+*   Re: code review, [Petre](http://arxiv.org/abs/1407.5648) found:
+    *   Requires domain knowledge to be useful
+    *   No point doing it at submission time
+*   But domain experts are scarce
+*   No incentives to review someone else's thesis project code
+*   At present, only sustainable within team projects
```

Comment:
Are you referring to the last bullet? The "findings" should refer only to the two sub-bullets, no?